core/arch/arm/kernel/user_ta.c

```diff
@@ -313,6 +313,14 @@ static TEE_Result ta_load(const TEE_UUID *uuid,
 		goto error_return;
 	}
 
+	if (ta_head->flags & TA_FLAG_MULTI_SESSION &&
```

Comment:
@jenswi-linaro  do you mean that we force the TA_FLAG_SINGLE_INSTANCE flag when TA_FLAG_INSTANCE_KEEP_ALIVE or TA_FLAG_MULTI_SESSION  is set?  if so,  i think it is a bit different from what the GP Internal Core API v1.1 describes.